packages/camera/lib/camera.dart

```diff
@@ -242,6 +244,12 @@ class CameraController extends ValueNotifier<CameraValue> {
   /// Whether to include audio when recording a video.
   final bool enableAudio;
 
+  // Torch
+  final bool enableTorch;
```

Comment:
nit: This and `enableAE` should explain more what these members symbolize, and should also be `/// ` to show that they're documentation and not implementation comments.

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc

Reply:
I added more detailled comments.